backend/src/main/kotlin/org/jetbrains/kotlinconf/backend/Sessionize.kt

```diff
@@ -8,14 +8,15 @@ import io.ktor.server.application.log
 import kotlinx.coroutines.DelicateCoroutinesApi
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.first
 import kotlinx.coroutines.launch
 import org.jetbrains.kotlinconf.Conference
 import org.jetbrains.kotlinconf.Session
 import org.jetbrains.kotlinconf.Speaker
 import java.util.concurrent.TimeUnit
 
-@Volatile
-private var conference: Conference? = null
+private val conference = MutableSharedFlow<Conference>()
```

Comment:
Shouldn't this be a `MutableStateFlow` so it always holds the latest value that was stored in it? With that we could make `getSessionizeData` non-suspending again, and just return `conference.value`.

Reply:
To get the initial state we need to suspend, so I'm not sure if it is possible to replace it

Reply:
Makes sense! I didn't look at the usage in detail before, but now I see that the API call to grab the data needs to wait for this. In this case, we should probably have a `replay = 1` set on this, to make sure that once a value is set, it's kept around?

Reply:
done, added test